### PR TITLE
Added current score display

### PR DESCRIPTION
**Pull request by michalStarski**
Ad. #29

### CODE REVIEW

**Comment by TiagoDanin** [comment]
@michalStarski avoid issues already assigned

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3647 @@
+{
```
Comment (TiagoDanin) [inline package-lock.json:1]: Pls remove `package-lock.json`
No use NPM, use Yarn.
`yarn install`
`yarn build`

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

**Comment by michalStarski** [comment]
Hey! I thought you wouldn't review my changes since the issue has been assigned to someone else. With that in mind i accidentally already deleted my fork so I had to create another one. Since I can't go back to this PR I've created another one with the same changes + requested changes to it. It's avaliable here:

#31

Please, take a look and sorry for any inconvenience,

**Comment by TiagoDanin** [comment]
> Hey! I thought you wouldn't review my changes since the issue has been assigned to someone else. With that in mind i accidentally already deleted my fork so I had to create another one. Since I can't go back to this PR I've created another one with the same changes + requested changes to it. It's avaliable here:
> 
> #31
> 
> Please, take a look and sorry for any inconvenience.
> 
> (Of course after review you can close this one)

No problem :)

**Pull request closed without merging**